tests/test_order_SolidLiquid.py

```diff
@@ -49,6 +50,30 @@ def test_identical_environments(self):
                 self.assertEqual(comp.cluster_sizes[0], len(positions))
                 npt.assert_array_equal(comp.num_connections, 12)
 
+    def test_l_zero(self):
```

Comment:
Tests need to be added for one or more nontrivial cases that enforce the conventions of which angles are phi/theta and m-indexing. Especially tests that leverage both even and odd parity. I arbitrarily suggest something like `l=6, m=2,3` with 1 neighbor that is at a non-zero value of phi, theta.

Reply:
I just wrote a general test that validates the entire calculation. Seems more reliable.